source/scss/clarendon.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.3) saturate(1.2);
+
+  &:after {
+    background: rgba(239, 239, 239, .8);
+    mix-blend-mode: soft-light;
+  }
+
+  &:before {
+    background: rgba(17, 129, 147, .23);
```

Comment:
This is causing a bit of a blue wash -- what do you think of trying an opacity of `.1` instead of `.23`?